src/utils/ChildRerenderer.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { ReactNode } from "react";
+
+interface Props {
+  children: ReactNode;
+}
+
+const symbol = Symbol("ChildRerenderer");
+
+export function ChildRerenderer({ children }: Props) {
```

Comment:
I think it should be `ChildrenRenderer` because we are rendering `children`. what do you think?
```suggestion
export function ChildrenRenderer({ children }: Props) {
```

Reply:
I have renamed the component's name to `ChildrenRerenderer`. Thanks! 👍  [4dae4bd](https://github.com/scottrippey/next-router-mock/pull/37/commits/4dae4bdf4ec1dc9d2bd362011ae466715217faa9)